src/libfsm/print/c.c

```diff
@@ -432,7 +432,9 @@ fsm_print_c(FILE *f, const struct fsm *fsm)
 		fprintf(f, "{\n");
 		fprintf(f, "\tconst char *p;\n");
 		fprintf(f, "\n");
-		fprintf(f, "\tassert(s != NULL);\n");
+		fprintf(f, "\tif (s == NULL) {\n");
+		fprintf(f, "\t\treturn -1;\n");
```

Comment:
Is this recoverable? If so, should it be distinct from the -1 `EOF` result?

Reply:
Undefined behavior it is! dc52d7c